sops/3scale-scaling.md

```diff
@@ -7,54 +7,96 @@ The purpose of this guide is to outline the existing scaling capabilities of 3Sc
-    - You can scale this component **horizontally** by adding more PODs
-        - `oc scale dc apicast-<staging or production> -n <ns-prefix>-3scale --replicas=<number-of-replicas>`
-    - You can scale this component **vertically** by deploying one worker for each CPU core available to the apicast process. This is controlled by the variable `APICAST_WORKERS`
-    - *Recommended HA --* Deploy a minimum of 2 PODs in different openshift nodes.
-1. System-Provider & System-Developer
-    - This component scales **horizontally** by adding more PODs
-        - `oc scale dc system-<provider or developer> -n <ns-prefix>-3scale --replicas=<number-of-replicas>`
-    - **Note :** As it depends on system-mysql, keep an eye open for common mysql problems.
-    - *Recommended HA --* Deploy 2 PODs
-1. System-Redis
-    - **Vertical** scaling of each node (core) redis run on: 
-        - Redis is a single threaded application, so each redis pod at most can use one core. A single redis pod does not scale with number of cores on the host node, or the number of host nodes.
-        - CPU Speed, RAM available splitting redis DMs to run on separate PODs on a multi-core node or across nodes
-1. Backend-Redis
-    - **Vertical** scaling of each node (core) redis run on: 
-        - Redis is a single threaded application, so each redis pod at most can use one core. A single redis pod does not scale with number of cores on the host node, or the number of host nodes.
-        - CPU Speed, RAM available splitting redis DMs to run on separate PODs on a multi-core node or across nodes
-1. System-memcache
-    - **Vertical** scaling of each node (core) redis run on: 
-        - Redis is a single threaded application, so each redis pod at most can use one core. A single redis pod does not scale with number of cores on the host node, or the number of host nodes.
-        - CPU Speed, RAM available splitting redis DMs to run on separate PODs on a multi-core node or across nodes
-1. System-Sidekiq 
-    - Due to the nature of the background jobs they don't usually require to be scaled, one POD should be enough, but both components can be scaled **horizonatally**.
-        - `oc scale dc system-sidekiq -n <ns-prefix>-3scale --replicas=<number-of-replicas>`
-    - These is also `RAILS_MAX_THREADS` which allows for **vertical** scaling, but is recommended to scale horizontally over this approach.
-    - *Recommended HA --* Deploy 2 PODs
-1. Backend-Cron
-    - There should be only one copy of this component running. If it fails, OpenShift will restart a new pod to replace it and it will pick-up where the previous pod ended and process remaining work.
-    - No need to scale this service
+### Backend-Listener
+#### Scalability:
+- Pod **anti-affinity** rule is important here
+- You can scale this component **horizontally** by adding more PODs
+    - `oc scale dc backend-listener -n <ns-prefix>-3scale --replicas=<number-of-replicas>`
+
+####Depends on:
```

Comment:
Not formatting correctly because I'm guessing missing space between #### and Depends.  The one the Apicast staging/prod section formats correctly.